Libraries/Opc.Ua.Server/Server/StandardServer.cs

```diff
@@ -3819,7 +3820,8 @@ protected override void OnServerStopping()
                             -= SessionChannelKeepAliveEvent;
                         m_serverInternal.SubscriptionManager.Shutdown();
                         m_serverInternal.SessionManager.Shutdown();
-                        m_serverInternal.NodeManager.Shutdown();
+                        m_serverInternal.NodeManager.ShutdownAsync()
+                            .AsTask().GetAwaiter().GetResult();
```

Comment:
async void would be better since this seems to be a event handler, if not we should make the method async